src/Tasks/FileIO/HashHelper.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+
+namespace Microsoft.Build.Tasks
+{
+    internal class HashHelper
```

Comment:
I think you should just move `HashHelper`'s stuff to `GetFileHash` and call it from `VerifyFileHash`.